GNUmakefile

```diff
@@ -7,6 +7,15 @@ default: build
 build: fmtcheck
 	go install
 
+setup:
+	go mod download
+	cd tools && GO111MODULE=on go install github.com/client9/misspell/cmd/misspell
+	cd tools && GO111MODULE=on go install github.com/katbyte/terrafmt
+	cd tools && GO111MODULE=on go install github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
https://golangci-lint.run/usage/install/#install-from-source

> Note: such go get installation aren't guaranteed to work. We recommend using binary installation.

Reply:
There are two options.

1. Install golangci-lint with the official script
1. Leave the installation to developers

## Install golangci-lint with the official script

https://golangci-lint.run/usage/install/#other-ci

```
$ curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.33.0
```

In that case, `curl` is required. And we should update the version of golangci-lint with [Renovate](https://docs.renovatebot.com/) or something.

Reply:
Oh, sorry.
I understood. https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module